Sources/TuistKit/Commands/Project/ProjectShowCommand.swift

```diff
@@ -0,0 +1,43 @@
+import ArgumentParser
+import Foundation
+import Path
+import TuistSupport
+
+struct ProjectShowCommand: AsyncParsableCommand {
+    static var configuration: CommandConfiguration {
+        CommandConfiguration(
+            commandName: "show",
+            _superCommandName: "project",
+            abstract: "Show information about the specified project. Use --web flag to open the project in the browser."
+        )
+    }
+
+    @Argument(
+        help: "The project to show. The full handle must be in the format of account-handle/project-handle.",
+        completion: .directory,
+        envKey: .projectShowFullHandle
+    )
+    var fullHandle: String?
```

Comment:
GitHub refers to `tuist/tuist` as `full_name` in their API – and otherwise repository. I think a pair of `full_handle` and `project` is the equivalent for us. I'm not sure why we need to bring the server semantics for this name.